From 6fbc14fc5440940179a04b6dec515f7ef46ed024 Mon Sep 17 00:00:00 2001
From: sarami_77 <sarami_77>
Date: Sun, 28 Jan 2024 13:53:24 +0900
Subject: [PATCH 2/2] 8v8

---
 EnemyRush/EnemyRush.ostw | 64 ++++++++++++++++++++--------------------
 1 file changed, 32 insertions(+), 32 deletions(-)

diff --git a/EnemyRush/EnemyRush.ostw b/EnemyRush/EnemyRush.ostw
index 6242e18..86a7e38 100644
--- a/EnemyRush/EnemyRush.ostw
+++ b/EnemyRush/EnemyRush.ostw
@@ -542,7 +542,7 @@ if(
 }
 
 // TAG EnemyBot/Config ----------
-define basicSpawnTimeBase: 10.0;
+define basicSpawnTimeBase: 10.0 * (8/12);
 define basicSpawnTimeScale: 1.0;
 
 define basicSpawnTimeMin: 
@@ -749,7 +749,7 @@ if(
 }
 
 // TAG EnemyBot/Asset ----------
-define maxCreateEnemyBotNum: 12;
+define maxCreateEnemyBotNum: 8;
 
 globalvar define sameHero!;
 
@@ -760,44 +760,44 @@ define enemyAssetList:
             // assetNo 0
             // basic
             [EnemyType.SubBoss,     RandomHero,         1],
-            [EnemyType.Elite,       RandomHero,         5],
-            [EnemyType.Minion,      RandomHero,         6]
+            [EnemyType.Elite,       RandomHero,         3],
+            [EnemyType.Minion,      RandomHero,         4]
         ],
         [   
             // assetNo 1
             // sub boss and minion
             [EnemyType.SubBoss,     RandomHero,         1],
-            [EnemyType.Minion,      RandomHero,         11]
+            [EnemyType.Minion,      RandomHero,         7]
         ],
         [   
             // assetNo 2
             // all elite
-            [EnemyType.Elite,       RandomHero,         4],
-            [EnemyType.Elite,       RandomHero,         4],
-            [EnemyType.Elite,       RandomHero,         4]
+            [EnemyType.Elite,       RandomHero,         3],
+            [EnemyType.Elite,       RandomHero,         3],
+            [EnemyType.Elite,       RandomHero,         2]
         ],
         [   
             // assetNo 3
             // all elites same hero
-            [EnemyType.Elite,       sameHero,           12]
+            [EnemyType.Elite,       sameHero,           8]
         ],
         [   
             // assetNo 4
             // half elite
-            [EnemyType.Elite,       RandomHero,         6],
-            [EnemyType.Minion,      RandomHero,         6]
+            [EnemyType.Elite,       RandomHero,         4],
+            [EnemyType.Minion,      RandomHero,         4]
         ],
         [   
             // assetNo 5
             // all minion
-            [EnemyType.Minion,      rva([FavoriteDamageHero(), FavoriteTankHero()]),   12]
+            [EnemyType.Minion,      rva([FavoriteDamageHero(), FavoriteTankHero()]),   8]
         ],
         [   
             // assetNo 6
             // double sub boss
             [EnemyType.SubBoss,     RandomHero,         1],
             [EnemyType.SubBoss,     RandomHero,         1],
-            [EnemyType.Minion,      RandomHero,         10]
+            [EnemyType.Minion,      RandomHero,         6]
         ],
         [   
             // assetNo 7
@@ -810,7 +810,7 @@ define enemyAssetList:
             // assetNo 8
             // damage boss with minion
             [EnemyType.Boss,        FavoriteDamageHero(),          1],
-            [EnemyType.Minion,      RandomHero,         11]
+            [EnemyType.Minion,      RandomHero,         7]
         ],
         // 9,
         [],
@@ -820,50 +820,50 @@ define enemyAssetList:
             // assetNo 10
             // Reinhardt army
             [EnemyType.SubBoss,     Hero.Reinhardt,     1],
-            [EnemyType.Elite,       Hero.Reinhardt,     5],
-            [EnemyType.Minion,      Hero.Reinhardt,     6]
+            [EnemyType.Elite,       Hero.Reinhardt,     3],
+            [EnemyType.Minion,      Hero.Reinhardt,     4]
         ],
         [   
             // assetNo 11
             // Soldier76 army
             [EnemyType.SubBoss,     Hero.Soldier76,     1],
-            [EnemyType.Elite,       Hero.Soldier76,     5],
-            [EnemyType.Minion,      Hero.Soldier76,     6]
+            [EnemyType.Elite,       Hero.Soldier76,     3],
+            [EnemyType.Minion,      Hero.Soldier76,     4]
         ],
         [   
             // assetNo 12
             // Pharah army
             [EnemyType.SubBoss,     Hero.Pharah,        1],
-            [EnemyType.Elite,       Hero.Pharah,        5],
-            [EnemyType.Minion,      Hero.Pharah,        6]
+            [EnemyType.Elite,       Hero.Pharah,        3],
+            [EnemyType.Minion,      Hero.Pharah,        4]
         ],
         [   
             // assetNo 13
             // Roadhog army
             [EnemyType.SubBoss,     Hero.Roadhog,       1],
-            [EnemyType.Elite,       Hero.Roadhog,       5],
-            [EnemyType.Minion,      Hero.Roadhog,       6]
+            [EnemyType.Elite,       Hero.Roadhog,       3],
+            [EnemyType.Minion,      Hero.Roadhog,       4]
         ],
         [   
             // assetNo 14
             // Bastion army
             [EnemyType.SubBoss,     Hero.Bastion,       1],
-            [EnemyType.Elite,       Hero.Bastion,       5],
-            [EnemyType.Minion,      Hero.Bastion,       6]
+            [EnemyType.Elite,       Hero.Bastion,       3],
+            [EnemyType.Minion,      Hero.Bastion,       4]
         ],
         [   
             // assetNo 15
             // gun and tank
             [EnemyType.Boss,        Hero.Widowmaker,    1],
             [EnemyType.SubBoss,     Hero.Widowmaker,    3],
-            [EnemyType.Minion,      FavoriteTankHero(), 8]   
+            [EnemyType.Minion,      FavoriteTankHero(), 4]   
         ],
         [
             // assetNo 16
             // gun and shield
             [EnemyType.SubBoss,     Hero.Mauga,       1],
-            [EnemyType.Elite,       Hero.Sigma,       5],
-            [EnemyType.Minion,      Hero.Bastion,     6] 
+            [EnemyType.Elite,       Hero.Sigma,       3],
+            [EnemyType.Minion,      Hero.Bastion,     4] 
         ],
         // 17,18,19,
         [],[],[],
@@ -873,8 +873,8 @@ define enemyAssetList:
             // assetNo 20
             // damage boss
             [EnemyType.Boss,        FavoriteDamageHero(),   1],
-            [EnemyType.Elite,       RandomHero,             5],
-            [EnemyType.Minion,      RandomHero,             6]
+            [EnemyType.Elite,       RandomHero,             3],
+            [EnemyType.Minion,      RandomHero,             4]
         ],
         [
             // assetNo 21
@@ -887,15 +887,15 @@ define enemyAssetList:
             // assetNo 22
             // tank boss
             [EnemyType.Boss,        FavoriteTankHero(),     1],
-            [EnemyType.Elite,       FavoriteDamageHero(),   5],
-            [EnemyType.Minion,      RandomHero,             6]
+            [EnemyType.Elite,       FavoriteDamageHero(),   3],
+            [EnemyType.Minion,      RandomHero,             4]
         ],
         [
             // assetNo 23
             // double damage boss
             [EnemyType.Boss,        FavoriteDamageHero(),   1],
             [EnemyType.Boss,        FavoriteDamageHero(),   1],
-            [EnemyType.Minion,      RandomHero,             10]
+            [EnemyType.Minion,      RandomHero,             6]
         ],
         // 24,25,26,27,28,29,
         [],[],[],[],[],[],
-- 
2.33.0.windows.2

